Change offset type from unsigned long to long long to support large file offsets.

Upstream-Status: Inappropriate No time for upstreaming

--- a/parser/parser.c
+++ b/parser/parser.c
@@ -464,7 +464,7 @@ static int parse_common_attributes(parsertype p, void *elem, struct img_type *i
 {
 	char seek_str[MAX_SEEK_STRING_SIZE];
 	const char *compressed, *encrypted;
-	unsigned long offset = 0;
+	long long offset = 0;
 
 	/*
 	 * GET_FIELD_STRING does not touch the passed string if it is not
